### PR TITLE
100% pep8 in the root dir

### PR DESCRIPTION
**Pull request by skyl**
better stopping point for master, I think.

### CODE REVIEW

File: bl/midi.py [hunk 1]
```diff
@@ -181,16 +188,19 @@ class MidiHandler(object):
 
     def __call__(self, message):
         """
-        Parse method and call method on self based on midi function. For example
-        if function is NOTEON_CHAN1, this will call our method noteon(), etc. If
-        a message has a channel as part of it's function, this will be the first
-        argument. After the first optional channel argument, remaining positional
-        arguments are passed to the method in the same order as specified in MIDI.
-        Not all MIDI functions need to be supplied or implemented in a subclass.
```
Comment (djfroofy) [inline bl/midi.py:189]: Needs some justification. I might clean up after pulling.

File: bl/player.py [hunk 2]
```diff
@@ -235,49 +253,58 @@ def _advance(self, last, schedule, event=None):
...
     """
-    A Conductor plays a score graphs which consists of nodes which each give
-    the duration in measures to play a node, a list of players (IPlayable) to start
-    and stop at the beginning and end of the duration and a list of trasitions
-    (other keys into) the graph to randomly transtition to.  The reserved key START
-    (equal to None) is for designating which node to start at.  The length of the
-    measure is determined by the supplied clock's (or the default global clock's)
+    A Conductor plays a score graphs which consists of nodes.
+    Each node has:
+
```
Comment (djfroofy) [inline bl/player.py:275]: List items should Have first word capitalized and maybe preceded by "*"

File: bl/player.py [hunk 3]
```diff
@@ -235,49 +253,58 @@ def _advance(self, last, schedule, event=None):
...
-                 'b': { 'players' : [player1, player3], 'duration': 1,
+                 'b': { 'players' : [player1, player3],
+                        'duration': 1,
                         'transitions' : ['a']} }
         conductor = Conductor(score)
         conductor.start()
 
-    In the above example, the conductor once started will play player1 and
-    player2 for 2 measures, then transition to itself or the next node 'b' with
-    50/50 chance of either. When node 'b' is stated player1 and player3 with play
```
Comment (djfroofy) [inline bl/player.py:273]: Justification ... first line

File: bl/recorder.py [hunk 4]
```diff
@@ -38,21 +41,22 @@ def record(self, event):
...
+            self._last_ticks = ticksper
         elapsed = ticks - self._last_ticks
         self._buffer.append((event, elapsed))
-        
+
     def latch(self, index=0):
         """
-        Return a completed loop. By default, this will return the last complete recorded
-        loop, otherwise you can return a past loop (up to 10) with index. For example, 
-        index=1 will return the loop before the last, ..., index=9, the 10th loop in the past.
```
Comment (djfroofy) [inline bl/recorder.py:51]: ... justification

File: bl/recorder.py [hunk 5]
```diff
@@ -25,10 +25,13 @@ def __init__(self, measures=1, clock=None, meter=None):
 
     def record(self, event):
         """
-        Record an event. If this the end of the loop duration, and the recorded events
-        create a loop different from the past loop, add to recorded stack.
```
Comment (djfroofy) [inline bl/recorder.py:29]: justification

File: bl/player.py [hunk 6]
```diff
@@ -622,13 +662,14 @@ def __init__(self, instr, notes, beats=16, clock=None, meter=None):
...
         if DEBUG:
-            log.msg('[StepSequencer.setVelocity] setting velocity at beat=%d to %d' %
-                    (beat, velocity))
+            log.msg('[StepSequencer.setVelocity] beat=%d to velocity=%d' %
+                                                    (beat, velocity))
         self.velocity[beat] = velocity
 
     def setStep(self, beat, note, on_off):
         if DEBUG:
-            log.msg('[StepSequencer.setStep] setting %dx%d=%d' % (beat, note, on_off))
```
Comment (djfroofy) [inline bl/player.py:631]: Is this pep-8, may indent to 1-char after paren above?

File: bl/midi.py (hunk 1 above)
Comment (skyl) [inline bl/midi.py:189]: I like it better with the line breaks on phrases.  But, I guess I don't really care.
It's your method.

It was a good exercise to break it up while reading the code/docs.

Comment (djfroofy) [inline bl/midi.py:189]: Yes, that's a good idea - breaking up by phrases.

However, in that case, I'd separate with two newlines.

Like this comment.

But you never know. Sometimes, you want to bust out a paragraph. Like this one,
as an example. And for a paragraph it reads better justified, I perceive.

Fur rilz.

File: bl/player.py (hunk 6 above)
Comment (skyl) [inline bl/player.py:631]: I don't know what you mean here.

File: bl/player.py [hunk 7]
```diff
@@ -235,49 +253,58 @@ def _advance(self, last, schedule, event=None):
...
         conductor = Conductor(score)
         conductor.start()
 
-    In the above example, the conductor once started will play player1 and
-    player2 for 2 measures, then transition to itself or the next node 'b' with
-    50/50 chance of either. When node 'b' is stated player1 and player3 with play
+    In the above example,
+    the conductor once started will play player1 and player2 for 2 measures,
+    then transition to itself or the next node 'b' with 50/50 chance of either.
+    When node 'b' is stated player1 and player3 with play
```
Comment (skyl) [inline bl/player.py:300]: *started *will

**Pull request merged**